tests/test_simple.py

```diff
@@ -4,25 +4,30 @@
 from dotenv import load_dotenv
 from llmtag.llm_label import get_note_label
 
-from .load_test_data import load_simulation_df1
+from .load_test_data import load_simulation_df1, load_labeled_df1, load_gpt_sim, load_gpt_sim_small
```

Comment:
what is your reasoning behind have three load functions instead of one which takes a filename input?